src/utils.ts

```diff
@@ -206,19 +263,19 @@ function findAncestorNs(doc, docSubsetXpath, namespaceResolver) {
   return returningNs;
 }
 
-function validateDigestValue(digest, expectedDigest) {
+export function validateDigestValue(digest, expectedDigest) {
   let buffer;
   let expectedBuffer;
 
-  const majorVersion = /^v(\d+)/.exec(process.version)[1];
+  const majorVersion = (/^v(\d+)/.exec(process.version) || [0, 0])[1];
 
   if (+majorVersion >= 6) {
     buffer = Buffer.from(digest, "base64");
     expectedBuffer = Buffer.from(expectedDigest, "base64");
   } else {
     // Compatibility with Node < 5.10.0
-    buffer = new Buffer(digest, "base64");
-    expectedBuffer = new Buffer(expectedDigest, "base64");
+    buffer = Buffer.from(digest, "base64");
+    expectedBuffer = Buffer.from(expectedDigest, "base64");
```

Comment:
I'm proposing to revert this or to remove it altogether. 

If we were to _keep_ this, it would trigger a runtime error, if running on Node 5. That is why there is a check for Node major version earlier in the code path; to determine if Buffer.from is present. As it is, both branches of the conditional are largely the same.

Reply:
I've made the change, but I'm also wondering the history of the lines below this with the comment "Compatibility with Node < 0.11.13" Does that need to be removed? Honestly, it looks like a reasonable check to me no matter the version of Node.